task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -1034,6 +1036,14 @@ def _handle_request(self, msg: ToSupervisor, log: FilteringBoundLogger):
         elif isinstance(msg, GetXComCount):
             len = self.client.xcoms.head(msg.dag_id, msg.run_id, msg.task_id, msg.key)
             resp = XComCountResponse(len=len)
+        elif isinstance(msg, GetXComSequenceItem):
```

Comment:
Could you add this to `task-sdk/tests/task_sdk/execution_time/test_supervisor.py::TestHandleRequest::test_handle_requests` too, please

Reply:
Add a couple of cases.